src/scenes/App/index.js

```diff
@@ -1,46 +1,38 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Reboot from 'material-ui/Reboot';
-import { connect } from 'react-redux';
-import { Debounce } from 'react-throttle';
-import { WindowResizeListener } from 'react-window-resize-listener';
-import { withStyles } from 'material-ui/styles';
+import { CssBaseline, withStyles } from 'material-ui';
 
 import styles from './styles';
 import AppRouter from './router';
 import GlobalHub from './globalHub';
 import Loader from './components/Loader/index';
-import appActions from '../../redux/App/actions';
 import BottomBar from '../../components/BottomBar/index';
-import NavBar from '../../components/NavBar/index';
 import CreateEvent from '../CreateEvent/index';
 import PendingTransactionsSnackbar from '../../components/PendingTransactionsSnackbar/index';
 import TransactionSentDialog from '../../components/TransactionSentDialog/index';
 import WalletUnlockDialog from '../../components/WalletUnlockDialog/index';
 import ErrorDialog from '../../components/ErrorDialog/index';
 
-const { toggleAll } = appActions;
 
-export class App extends React.PureComponent {
+@withStyles(styles)
+export default class App extends React.PureComponent {
+  static propTypes = {
+    match: PropTypes.object.isRequired,
+    classes: PropTypes.object.isRequired,
+    langHandler: PropTypes.func.isRequired,
+    txReturn: PropTypes.object,
+  }
+
+  static defaultProps = {
+    txReturn: undefined,
+  }
+
   render() {
-    const {
-      classes,
-      txReturn,
-    } = this.props;
-    const { url } = this.props.match;
+    const { classes, txReturn, match: { url } } = this.props;
 
     return (
       <div className={classes.root}>
-        <Reboot />
-        <Debounce time="1000" handler="onResize">
-          <WindowResizeListener
-            onResize={(windowSize) =>
-              this.props.toggleAll(
-                windowSize.windowWidth,
-                windowSize.windowHeight
-              )}
-          />
-        </Debounce>
+        <CssBaseline />
```

Comment:
This needed to be changed because `<Reboot />` is deprecated and replaced by this. See here https://material-ui-next.com/style/css-baseline/

Reply:
Also, not sure if we really need this since we're using `normalize`